README.rst

```diff
@@ -34,9 +34,12 @@ Maintained Major Branches
 ====== ==================== ===============
 Major  Initial release date Supported until
 ====== ==================== ===============
-v1.x.x v1.4.0_: 24/06/2016  TBD
+v1.x.x v1.4.0_: 24/06/2016  Fri 08/12/2017
+====== ==================== ===============
+v2.x.x v2.0.1_: 11/09/2017  TBD
 ====== ==================== ===============
 .. _v1.4.0: https://github.com/sociomantic-tsunami/makd/releases/tag/v1.4.0
+.. _v2.0.1: https://github.com/sociomantic-tsunami/makd/releases/tag/v2.0.1
```

Comment:
There is an inconsistency in here, the end of support for v1 is 08/12/2017 but you are putting the release of v2 on 11/09/2017. I think you got the date from the release of v2.0.0, not v2.0.1, and, even when broken, I think the real initial release should be put here, i.e. v2.0.0.

Reply:
Actually I wanted to align it on a week's boundary. But yeah, this is a weird mix, will fix.